chart/README.MD

```diff
@@ -0,0 +1,24 @@
+# ToCloud9 Helm chart
+
+This chart will deploy a ToCloud9 cluster using **AzerothCore** as backend.
+
+> **WARNING:_** 
+Pre-built images in this chart values won't be actively maintained, build your own ones if you want to be up to date with AzerothCore releases.
+
+## How to use
+
+1. Check the values to set a storage class for persistency and load-balancer domain.
+2. Deploy the chart
+3. Most services won't start as long as the databases tables aren't initialized with the [AzerothCore sql scripts](https://github.com/azerothcore/azerothcore-wotlk/tree/master/data/sql/base) and [ToCloud9 modifications](../doc/RunNonDockerWinWSLAzerothCore.md#setup--run). Databases and user are already created during mysql deployment.
+4. Gameserver will require its `/data` volume to be provisionned with files to be [downloaded](https://www.azerothcore.org/wiki/server-setup#download) or [extracted](https://www.azerothcore.org/wiki/server-setup#extractors).
```

Comment:
Right now I use a NFS provider with my NAS to get PVC provisionned. To provision the data, I downloaded the file and extracted it to the PV folder on the NAS side. It really comes down to what user will choose as storage backend. We can always set an init container that would do this at startup, but it could take a while to perform. 

Are those files read only during server runtime? Are there any lock or anything put on file or could I leverage a ReadWriteMany PV to share the folder between gameserver workloads?

Reply:
I think it's read only. So ReadWriteMany or ReadOnlyMany should work.